packages/cursorless-engine/src/processTargets/targets/PlainTarget.ts

```diff
@@ -20,5 +20,10 @@ export default class PlainTarget extends BaseTarget {
   getTrailingDelimiterTarget = () => undefined;
   getRemovalRange = () => this.contentRange;
 
-  protected getCloneParameters = () => this.state;
+  protected getCloneParameters() {
+    return {
+      ...this.state,
+      isToken: this.isToken,
```

Comment:
We really gotta figure out a way to make the clone parameters have a stronger type.  Forgetting to include custom parameters is something that has hit us multiple times